src/components/app-wrapper/metadata-helpers/__tests__/normalization.spec.ts

```diff
@@ -705,4 +719,44 @@ describe('normalization', () => {
             })
         })
     })
+
+    describe('LegendSetMetadataItem', () => {
+        it('should normalize LegendSetMetadataItem by returning it as-is', () => {
+            const input = {
+                id: 'legend1',
+                name: 'Test Legend Set',
+                legends: [
+                    {
+                        id: 'l1',
+                        name: 'Legend 1',
+                        startValue: 0,
+                        endValue: 10,
+                    },
+                ],
+            }
+            const result = normalizeMetadataInputItem(input as any)
+
+            expect(result).toEqual(input)
+        })
+    })
+    it('should extract required properties from LegendSet', () => {
+        const input = {
+            id: 'legend1',
+            name: 'Test Legend Set',
+            legends: [
+                { id: 'l1', name: 'Legend 1', startValue: 0, endValue: 10 },
+            ],
+            code: 'TEST_CODE',
+            displayName: 'Display Name',
+            someOtherProperty: 'should be omitted',
+        }
+
+        const result = normalizeLegendSetMetadataItem(input as any)
+
+        expect(result).toEqual({
+            id: input.id,
+            name: input.name,
```

Comment:
Shouldn't `displayName` be used instead to get the potential translations?
Or is `name` anyway containing the translated version based on the current user's preferences?